source/test/F0.Minesweeper.Logic.Tests/LocationTest.cs

```diff
@@ -0,0 +1,35 @@
+using FluentAssertions;
+using Xunit;
+
+namespace F0.Minesweeper.Logic.Tests
+{
+	public class LocationTest
+	{
+		[Fact]
+		public void NewLocation_XParameterEqualsX()
+		{
+			uint testValue = 5;
+			var locationUnderTest = new Location(testValue, 7);
+
+			locationUnderTest.X.Should().Be(testValue);
+		}
+
+		[Fact]
+		public void NewLocation_YParameterEqualsY()
+		{
+			uint testValue = 7;
+			var locationUnderTest = new Location(5, testValue);
+
+			locationUnderTest.Y.Should().Be(testValue);
+		}
+
+		[Fact]
+		public void NewLocation_MaxIntParameter()
```

Comment:
If we have this test I believe then we should have it for both the x and the y axis

Reply:
Done. I made another test for y and renamed this one.